Faraday_Proxy_Tools/FaradayIO/telemetryparser.py

```diff
@@ -140,96 +140,102 @@ def UnpackPacket_3(self, packet, debug = False):
-        del telemetryList[3] # Destination callsign length index - 1
-        del telemetryList[31] # N/A Byte is not needed
+        #del telemetryList[1] # Source callsign length index
+        #del telemetryList[3] # Destination callsign length index - 1
+        #del telemetryList[31] # N/A Byte is not needed
         telemetryList.append(time.time())
         telemetryList = tuple(telemetryList)
+        print repr(telemetryList)
 
 
 
 
         dictionaryData = {  'SOURCECALLSIGN': str(telemetryList[0]),
-                            'SOURCEID': int(telemetryList[1]),
-                            'DESTINATIONCALLSIGN': telemetryList[2],
-                            'DESTINATIONID': telemetryList[3],
-                            'RTCSEC': telemetryList[4],
-                            'RTCMIN': telemetryList[5],
-                            'RTCHOUR': telemetryList[6],
-                            'RTCDAY': telemetryList[7],
-                            'RTCDOW': telemetryList[8],
-                            'RTCMONTH': telemetryList[9],
-                            'RTCYEAR': telemetryList[10],
-                            'GPSLATITUDE': telemetryList[11],
-                            'GPSLATITUDEDIR': telemetryList[12],
-                            'GPSLONGITUDE': telemetryList[13],
-                            'GPSLONGITUDEDIR': telemetryList[14],
-                            'GPSALTITUDE': telemetryList[15],
-                            'GPSALTITUDEUNITS': telemetryList[16],
-                            'GPSSPEED': telemetryList[17],
-                            'GPSFIX': telemetryList[18],
-                            'GPSHDOP': telemetryList[19],
-                            'GPIOSTATE': telemetryList[20],
-                            'RFSTATE': telemetryList[21],
-                            'ADC0': telemetryList[22],
-                            'ADC1': telemetryList[23],
-                            'ADC2': telemetryList[24],
-                            'ADC3': telemetryList[25],
-                            'ADC4': telemetryList[26],
-                            'ADC5': telemetryList[27],
-                            'ADC6': telemetryList[28],
-                            'BOARDTEMP': telemetryList[29],
-                            'ADC8': telemetryList[30],
-                            'HABTIMERSTATE': telemetryList[31],
-                            'HABCUTDOWNSTATE': telemetryList[32],
-                            'HABTRIGGERTIME': telemetryList[33],
-                            'HABTIMER': telemetryList[34],
-                            'EPOCH': telemetryList[35]
+                            'SOURCECALLSIGNLEN': str(telemetryList[1]),
+                            'SOURCEID': int(telemetryList[2]),
+                            'DESTINATIONCALLSIGN': telemetryList[3],
+                            'DESTINATIONCALLSIGNLEN': telemetryList[4],
+                            'DESTINATIONID': telemetryList[5],
```

Comment:
OK you previously would delete index items 1 and 3. Adding it back them would in-fact change the indexing when parsing into a dictionary. These changes would add two to the index for items afterwards which appears correct. No longer deleting item 31 would push any item afterwards upwards by three which appears correct.